api/v1/cluster_webhook.go

```diff
@@ -1431,26 +1440,46 @@ func (r *Cluster) validateMinSyncReplicas() field.ErrorList {
 }
 
 func (r *Cluster) validateStorageSize() field.ErrorList {
-	return validateStorageConfigurationSize("Storage", r.Spec.StorageConfiguration)
+	return validateStorageConfigurationSize(*field.NewPath("Storage"), r.Spec.StorageConfiguration)
```

Comment:
Are we sure the path is `Storage` here? I think it's more like `field.NewPath("spec", "storage")`.
And no, this is not really about this patch, but it's better to fix it since we're changing this line of code.

Reply:
Was looking at this. I see the same problem in other parts.
So, delved into the function. I see this within the function...
```
    field.NewPath("spec", structPath.String(), "size")
```